properties/data/auto-starting-position-2025.md

```diff
@@ -0,0 +1,16 @@
+# auto-starting-position-2025
```

Comment:
please reformat this file so that it more closely matches the specification of the 2024 equivalent (which was updated through a recently merged pr)